conf/variant/arp/bblayers.conf.sample

```diff
@@ -8,4 +8,5 @@ BBLAYERS += "\
     ${BSPDIR}/sources/meta-pelux/meta-intel-extras  \
```

Comment:
could you please sort them out somehow in a logical way?

Reply:
Done!